librz/bin/pdb/pdb_downloader.c

```diff
@@ -23,8 +23,13 @@ static bool download_and_write(SPDBDownloaderOpt *opt, const char *file) {
 		free(dir);
 		return false;
 	}
-	char *url = rz_str_newf("%s/%s/%s/%s", opt->symbol_server, opt->dbg_file, opt->guid, file);
```

Comment:
1. Could you please also extract PDB downloading fixes into a separate PR too?
2. Please also rebase on top of the latest `dev`.